app/javascript/react/components/ReviewTile.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const ReviewTile = (props) => {
+  return(
+    <div className="callout cell large-12">
+      <li>{props.name}</li>
+      <li> {props.body}</li>
+      <li> {props.rating}</li>
+    </div>
```

Comment:
this is fine, but I have a feeling that showing these as `li` tags doesnt look super pretty